cmd/dbseed/main.go

```diff
@@ -242,4 +254,7 @@ func (s *Seeder) SeedData() {
 			},
 		},
 	)
+
+	s.createEmailRecipientModelDatesChanged("one@test.gov")
+	s.createEmailRecipientModelDatesChanged("two@test.gov")
```

Comment:
Are these wrapers (and associated store methods) still needed now that we're configuring recipients by env vars?